src/app/web-app.component.ts

```diff
@@ -113,6 +126,141 @@ export class WebAppComponent implements OnInit {
         verticalPosition: 'top'
       });
     });
-  }
 
+    // Keyboard Shortcuts Listener
+    this.unlisten = this.keyboardShortcuts.listen(
```

Comment:
Is it possible to shift this code to a separate file or a better way to implement this?

Reply:
I tried putting in a different file but it won't work as the shortcuts need to be implemented globally and has to render with the web-app component so can't move them also this is the only way as each shortcut has to be assigned different functionality, we've to write them individually and this was the shortest implemetation i could come up with.

Reply:
Alright, we can come back to this later.